--- server.clean.js	2025-01-03 00:09:20.000000000 +0000
+++ server.js	2025-01-03 00:20:47.000000000 +0000
@@ -43938,6 +43938,8 @@
                 if ("TLSSocket" != socketConstructor) return next("Unknown protocol");
                 protocol = "https://";
             }
+            // override protocol if behind reverse proxy
+            if ("https" == req.headers["x-forwarded-proto"]) protocol = "https://";
             var serverUrl = encodeURIComponent(protocol + req.headers.host), sep = webUILocation.includes("?") ? "&" : "?", location = webUILocation + sep + "streamingServer=" + serverUrl;
             res.writeHead(307, {
                 location: location
